content/articles/payment-methods.markdown

```diff
@@ -0,0 +1,34 @@
+---
+title: Payment methods
+excerpt: Payment methods supported by DNSimple.
+categories:
+- DNSimple
+---
+
+# Payment Methods
+
+DNSimple only supports two payment methods:
```

Comment:
~~only~~

_only_ makes this sounds like if it's a bad thing.

Reply:
👍